bundles/org.eclipse.e4.ui.model.workbench/src/org/eclipse/e4/ui/model/application/ui/basic/impl/PartImpl.java

```diff
@@ -1334,8 +1334,9 @@ public Object eInvoke(int operationID, EList<?> arguments) throws InvocationTarg
 
 	/**
 	 * <!-- begin-user-doc -->
+	 * If the method {@link #toStringGen()} below changes, please ensure that this is updated.
 	 * <!-- end-user-doc -->
-	 * @generated
+	 * @generated NOT
 	 */
```

Comment:
The `@generated NOT` ensures that this method is not overwritten during regeneration.